.flake8

```diff
@@ -37,7 +37,7 @@ extend-exclude =
 # IMPORTANT: avoid using ignore option, always use extend-ignore instead
 # Completely and unconditionally ignore the following errors:
 extend-ignore =
-  E203,  # skipped because it annoys black
```

Comment:
I like the original better, it's honest.

Also, fixed by #910

Reply:
@relrod Is this a fair compromise?